### ISSUE TITLE
Add fancier plugin select

### ISSUE THREAD

**Issue by pinussilvestrus** [pinussilvestrus/mopl#4]
__What should we do?__

Currently, the script user has to type a number to select a plugin.

![image](https://user-images.githubusercontent.com/9433996/163398801-686e7285-18dc-4f06-b76c-18b26e4ec973.png)

That should be solved user-friendlier, e.g. by making the selection keyboard navigatable. 

__Why should we do it?__

Improve usability.
